packages/frontend/src/routes/SignMessageWrapper.js

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import SignMessageInvalid from '../components/sign-message/SignMessageInvalid';
+import SignMessageSummaryWrapper from '../components/sign-message/SignMessageSummaryWrapper';
+import { redirectTo } from '../redux/actions/account';
+import { selectAccountUrlCallbackUrl, selectAccountUrlMessage, selectAccountUrlMeta } from '../redux/slices/account';
+import {
+    selectAvailableAccounts,
+    selectAvailableAccountsIsLoading,
+} from '../redux/slices/availableAccounts';
+import {
+    addQueryParams,
+    handleSignMessage,
+    selectSignMessageStatus,
+    selectSignedRequest,
+    SIGN_MESSAGE_STATUS,
+} from '../redux/slices/signMessage';
+import { isUrlNotJavascriptProtocol } from '../utils/helper-api';
+
+export function SignMessageWrapper() {
+    const dispatch = useDispatch();
+
+    const signCallbackUrl = useSelector(selectAccountUrlCallbackUrl);
+    const meta = useSelector(selectAccountUrlMeta);
```

Comment:
nit: While not enforced yet, we try to define selectors by simply removing the `select` part. E.g.:

`const accountUrlMeta = useSelector(selectAccountUrlMeta);`